client/python/examples/queues.py

```diff
@@ -9,11 +9,9 @@
 from armada_client.permissions import Permissions, Subject
 
 
-def creating_full_queues_example(client, queue):
+def create_queue_request(client, queue):
```

Comment:
any reason we aren't putting types in here, like we do for the code? (throughout)

Reply:
Python is a dynamic language. I just feel that we do not want to enforce the typing standards we set for Python Armada Client on the people that use it.

Reply:
I can see that, but type hints don't break non-type hinted code. Either way, I'll trust your judgement here.